commands/cartridge.go

```diff
@@ -45,4 +46,8 @@ func setLogLevel() {
 	if projectCtx.Verbose {
 		log.SetLevel(log.DebugLevel)
 	}
+
+	if projectCtx.Debug {
```

Comment:
Hm... Is it possible to specify `--verbose --debug`?

I think if you want to support several log levels you should add `--log-level=debug/verbose`

Reply:
I'm sure that using `--debug` is more convenient than `--log-level debug`.

Now there are three log level flags:

* `--verbose` prints additional messages (`log.DebugLevel`), it makes running commands more **verbose for user**;
* `--debug` is used for debug (`log.TraceLevel`). It doesn't remove tmp build directory and makes running commands more **verbose for developer**;
* `--quiet` is used to silence build command output (for example, `tarantoolctl rocks make` output), that makes commands more **convenient to use**.

For example, I use `--quiet --debug` to keep tmp build directory and silence build commands on packing debug.

If `--verbose --debug` is specified, the `--debug` level will be used because it includes `--verbose`.

Reply:
`--verbose --debug`
I'm not sure that this combination is appropriate.

Reply:
It can be a mistake, but I don't see anything bad here - this two modes don't exclude each other.

Reply:
It contains an implicit logic. One option affect another. I'm not sure that's good idea to allow enable both at one time.

Reply:
Decided do it later

Reply:
Added it to the checklist in #242